### ISSUE TITLE
Conflito com o módulo Pagar.me

### ISSUE THREAD

**Issue by fredpiuma** [FrenetGatewaydeFretes/frenet-magento#10]
Olá pessoal.

Não consigo utilizar esta versão do módulo de vocês em conjunto com o módulo da Pagar.me.

O corre um erro dizendo que alguma função da biblioteca GuzzleHttp não existe.

Verifiquei que a Pagar.me também utiliza esta bilioteca.

Existe alguma forma de instalar o módulo de vocês via composer? Acredito que isso resolveria esse conflito.

**Comment by andreyepcomm** [comment]
Sim, ocorre o erro quando tenta utiliza os dois módulos com a versão do php 7

**Comment by valiati100** [comment]
Pra resolver, basta editar o arquivo composer.json da raiz(que pertence ao modulo pagarme) e alterar a versão da biblioteca 'guzzlehttp/guzzle' para '^7.0'.

Depois, fazer um backup da pasta vendor e remove-la.

Executar o comando 'composer install'.

Lembrando que este ajuste foi testado em um cenário onde o módulo da pagarme foi instalado de forma manual, conforme eles explicam na documentação.

É de sumo importância fazer um backup completo antes de executar qualquer comando.
